test/dlc_tests/eks/mxnet/training/test_eks_mxnet_training.py

```diff
@@ -102,7 +102,7 @@ def test_eks_mxnet_dgl_single_node_training(mxnet_training, py3_only):
     cpu_limit = str(int(cpu_limit) / 2)
 
     if "gpu" in mxnet_training:
-        if "cu110" in mxnet_training:
+        if "cu112" in mxnet_training:
```

Comment:
Is the statement that this test shouldn't be run on cu110 as well still true? Also what about future versions of cuda?

If you're just trying to check for cuda 11, you could just change the string to `if "cu11" in mxnet_training` for simplicity.

If you need a specific version of cuda to check for, you can update this logic to use the `get_cuda_version_from_tag` utility in `test_utils/__init__.py` - then construct the version as `11.x` and use `Version` from `packaging.version` to compare versions ( see comment above about comparing framework versions).

You can also create a little utility function to do this since it seems like this is done in a few places.

Also please add a `# TODO: remove this when DGL supports cuda 11` comment, so that our text editors will highlight this

Reply:
we want to check for cuda version > 11.1 as they are not supported